src/services/getRepeatableSetEntriesConfiguration.ts

```diff
@@ -0,0 +1,25 @@
+import { FormTypes } from '@oneblink/types'
+import { RepeatableSetElement } from '@oneblink/types/typescript/forms'
+import { FormSubmissionModel } from '../types/form'
+import { submissionService } from '@oneblink/sdk-core'
+
+export default function getRepeatableSetEntriesConfiguration(
+  setEntries: RepeatableSetElement['minSetEntries'],
+  elements: FormTypes.FormElement[],
+  formSubmissionModel: FormSubmissionModel,
+) {
+  if (typeof setEntries === 'object') {
```

Comment:
When you check for `object` type, best to also check it is `!== null` as `null` is classified as `object` type. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof

Reply:
Obviously this shouldn't be null anyway, but its still a good idea when writing these kinds of type guards.